File: Chapter2/2.3/2.3.3-Xu Xiake in Henan Province/2.3.3-Xu Xiake in Henan Province.cpp
#include <iostream>
using namespace std;

int main(){
    int t,a1,a2,a3,a4,count;
    cin>>t;
    for (int i;i<t;i++){
        count=4;
        cin>>a1>>a2>>a3>>a4;
        count = a1==0?count-1:count;
        count = a2==0?count-1:count;
        count = a3==0?count-1:count;
        count = a4==0?count-1:count;
        if (count==0) cout<<"Typically Otaku"<<endl;
        else if (count==1) cout<<"Eye-opener"<<endl;
        else if (count==2) cout<<"Young Traveller"<<endl;
        else if (count==3) cout<<"Excellent Traveller"<<endl;
        else if (count==4) cout<<"Contemporary Xu Xiake"<<endl;
    }
}

File: Chapter2/2.3/2.3.3-Xu Xiake in Henan Province/2.3.3-Xu Xiake in Henan Province.py
t=int(input())
for i in range(t):
    As=list(input().split())
    count=4-As.count('0')
    if count==0:
        print('Typically Otaku')
    elif count==1:
        print('Eye-opener')
    elif count==2:
        print('Young Traveller')
    elif count==3:
        print('Excellent Traveller')
    else:
        print('Contemporary Xu Xiake')

